db/migrations/00002_create_nodes_table.sql

```diff
@@ -5,7 +5,8 @@ CREATE TABLE nodes (
   genesis_block VARCHAR(66),
   network_id    VARCHAR,
   node_id       VARCHAR(128),
-  CONSTRAINT node_uc UNIQUE (genesis_block, network_id, node_id)
+  chain_id      INTEGER,
+  CONSTRAINT node_uc UNIQUE (genesis_block, network_id, node_id, chain_id)
```

Comment:
chain_id needs to be added as a separate migrations so that we can apply it on the current database schema e.g. https://github.com/vulcanize/ipld-eth-indexer/blob/master/db/migrations/00012_add_chain_id_to_nodes.sql

Reply:
good point, will do. But here https://github.com/vulcanize/ipld-eth-indexer/blob/master/db/migrations/00012_add_chain_id_to_nodes.sql#L14 I think it should be "ADD COLUMN" not "DROP"?

Reply:
No the DROP is correct, those is the `down` migrations which are reversing the `up`

Reply:
It's reversing https://github.com/vulcanize/ipld-eth-indexer/blob/master/db/migrations/00012_add_chain_id_to_nodes.sql#L9

Reply:
@i-norden fixed it

Reply:
LGTM